src/main/java/gov/cms/madie/madieqdmservice/config/CqlElmTranslatorClientConfig.java

```diff
@@ -0,0 +1,23 @@
+package gov.cms.madie.madieqdmservice.config;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Getter
+@Configuration
+public class CqlElmTranslatorClientConfig {
+
+  @Value("${madie.cql-elm.service.base-url}")
+  private String baseUrl;
+
+  @Value("${madie.cql-elm.service.cql-elm-Url}")
+  private String cqlElmUri;
```

Comment:
Nitpick: `Url` starts with a capital. The associated parameter is `Uri`.

Reply:
updated it.